subsys/CMakeLists.txt

```diff
@@ -20,3 +20,4 @@ add_subdirectory(power)
 add_subdirectory(stats)
 add_subdirectory(testsuite)
 add_subdirectory_if_kconfig(jwt)
+add_subdirectory(can)
```

Comment:
I wonder if "canbus" would be better name for the subsystem, the word "can" is quite short and ambiguous.

Reply:
Changed to canbus. The name of the subsystem is now "CAN-Bus subsystem"